app/src/main/java/org/simple/clinic/widgets/qrcodescanner/QrCodeScannerView.kt

```diff
@@ -81,6 +86,7 @@ constructor(
     }, ContextCompat.getMainExecutor(context))
   }
 
+  @SuppressLint("ClickableViewAccessibility")
```

Comment:
What exactly is this suppressing? 🤔

Reply:
When we implement `onTouchListener` without a `performClick` action Android platform shows it as a warning since it can have an on-click listener separately. Since in this scenario we are sure preview view won't have an on-click listener I have suppressed that warning.